webviz_subsurface/ert_jobs/jobs.py

```diff
@@ -56,7 +56,7 @@ def _get_module_variable_if_exists(
     plugin_name="webviz-subsurface"
 )
 def job_documentation(job_name: str) -> Optional[dict]:
-    webviz_subsurface_jobs = set(installable_jobs().keys())
+    webviz_subsurface_jobs = set(installable_jobs().data.keys())
```

Comment:
With this change `pylint` CI gives this:
```
webviz_subsurface/ert_jobs/jobs.py:59:33: E1101: Instance of 'dict' has no 'data' member (no-member)
```
Is this a false positive that should be ignored, e.g. because the `ert` decorator does something magic?

@berland @lindjoha

Reply:
That is what I have assumed. I have added a pylint disable in my PR

Reply:
Ok, I can do some small changes to this PR and then we merge it. I must I don't understand what the ERT decorator does. What is the type of what is returned by `installable_jobs()`? Apparently not a `dict` anymore. :thinking:

Reply:
I think it is some kind of ert plugin object.

Reply:
The decorator apparently turns it into `<class 'ert_shared.plugins.plugin_response.PluginResponse'>`.